src/main/java/io/github/thebusybiscuit/slimefun4/integrations/IntegrationsManager.java

```diff
@@ -281,4 +288,7 @@ public boolean isChestTerminalInstalled() {
         return isChestTerminalInstalled;
     }
 
+    public boolean isOrebfuscatorInstalled() {
+		return isOrebfuscatorInstalled;
```

Comment:
I think there are tab characters in this line, we use spaces exclusively on this project.

Reply:
Oh, sry, I will fix that